src/shared/firewall-util-nft.c

```diff
@@ -0,0 +1,853 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
```

Comment:
This commit msg should probably contain "Fixes: #13307" somewhere, to link up the commit with the issue in a way github understands